MODERATION_POLICY.md

```diff
@@ -0,0 +1,297 @@
+  by the TSC) are subject to immediate Blocking.
+* Issues, pull requests, discussions, and comments that are spam (job posting,
+  service advertising, etc.) are subject to immediate moderation.
+* Issues, pull requests, discussions, and comments that are believed to be
+  LLM-generated (e.g a PR coming from a new contributor changing a single file
+  without clear motivation) should be closed with a comment such as "It seems
+  you are using a LLM, please stop, this is not bringing any value and is
+  wasting our time. If you are not using one, please read and follow our
+  contributing guidelines." Report the user to the moderation repository so they
+  get blocked if they do it again.
+* Collaborators may use the Hide feature in the GitHub interface for off-topic
+  posts by non-Collaborators.
+* TSC members can delete any issues or comments posted by accounts that have
+  been deleted by GitHub. These accounts show up in the GitHub interface as
+  user `ghost`. There is no need to screenshot or document these deletions.
+
+There are a few examples of moderating non-Collaborator posts:
+
+Scenario 1:
+ * A non-Collaborator posts a comment that indicates that they are a bot.
+ * A collaborator sees the post and hides it.
+ * No further action is necessary.
+
+Scenario 2:
+ * A non-Collaborator posts a comment that is against the Code of Conduct.
+ * A Collaborator sees the comment and asks the author to edit it.
+ * The author refuses to edit their comment.
+ * The Collaborator deletes the comment and posts an issue in the moderation
+   repository explaining their actions.
+
+Scenario 3:
+ * A non-Collaborator opens a pull request with comments indicating they are a
+   bot.
+ * A Collaborator sees that pull requests, closes it, deletes the comments
+   and posts an issue in the moderation repository explaining their actions.
+ * A TSC member sees the issue and decides to block the user from the
+   organization.
+
+Scenario 4:
+ * A non-Collaborator posts a comment on an old commit that is against the
+   Code of Conduct.
+ * A Collaborator sees the comment, takes a screenshot, and deletes it.
+ * The Collaborator posts an issue in the moderation repository explaining
+   their actions.
+
+### Temporary Interaction Limits
+
+The TSC may, at their discretion, choose to enable [GitHub's
+Temporary Interaction Limits][] on any GitHub repository in the webpack GitHub
+Organization.
+
+Any Collaborator may request that the TSC enable the Temporary
+Interaction Limits by posting an issue to the moderation repository. If the
+TSC chooses not to do so, then a comment explaining why that decision was made
+must be added to the moderation repository thread.
+
+### Temporary and Indefinite Blocks
+
+A Temporary Block is time limited, with the timeframe decided on by the Moderation
+Team at the time of issuing, depending on the severity of the issue. Recommended
+default options are 24-hour, 48-hour, and 7-day periods.
```

Comment:
dont use reccomend, just state it could be 24, 48hours, 7-day or any limit set by a collaborator. And that the collaborator must notify the length of the ban.

Reply:
Feel free to propose changes :)